src/ArduPilotPlugin.cc

```diff
@@ -84,14 +86,14 @@ class Control
   public: Control()
   {
     // most of these coefficients are not used yet.
-    this->rotorVelocitySlowdownSim = this->kDefaultRotorVelocitySlowdownSim;
-    this->frequencyCutoff = this->kDefaultFrequencyCutoff;
-    this->samplingRate = this->kDefaultSamplingRate;
+    this->rotorVelocitySlowdownSim = kDefaultRotorVelocitySlowdownSim;
+    this->frequencyCutoff = kDefaultFrequencyCutoff;
+    this->samplingRate = kDefaultSamplingRate;
```

Comment:
We can remove these completely. I think they are residual from aspects of the RotorS model the original plugin was adapted from. Not currently used in any code.